packages/lb-components/src/components/pointCloudView/hooks/usePointCloudViews.ts

```diff
@@ -571,6 +904,23 @@ export const usePointCloudViews = () => {
     viewUpdateBox(newPolygon, originPolygon, PointCloudView.Back);
   };
 
+  const topViewUpdatePoint = (updatePoint: IPointUnit, size: ISize) => {
```

Comment:
why not unify with viewUpdatePoint?

Reply:
First of all this two functions were related to "topViewUpdateBox" and "viewUpdateBox". 
Btw the logic in these two functions are different,  in"topViewUpdatePoint"  you can generate the new sphereParams just by updatedPoint, but in "viewUpdateBox" you will need both updatedPoint and originPoint these two params, cause the xAxis and yAxis in "topview" are different from that in "sideview" or "backview".